ui/lib/pki/addon/templates/roles/role/details.hbs

```diff
@@ -1 +1 @@
-roles.role.details
\ No newline at end of file
+<Page::Roles::Role::DetailsPage @role={{this.model}} />
```

Comment:
Open to feedback on this component name/structure. It's long, but assuming we have a page component for each page, it reflects the route structure

Reply:
I'm personally not a fan of the all the directories. I'm partial to a dasherized format like `pki-role-details` or `role-details` since it is in the pki engine, but that might not be everyone's favorite. We do seem to favor directories. I find it tougher to search when there might be many files named `DetailsPage` in different directories.

Reply:
Yeah. One nest down might be ok but this is a bit superfluous. What do you think at least about nesting it under `<Page::` ?

Reply:
Ya I think if you want to establish a pattern in the pki engine where page components will live that looks good to me.